TUM Campus App/NewsManager.swift

```diff
@@ -72,6 +72,14 @@ class NewsManager: Manager {
         }
     }
     
+    func getNextUpcomingNews() -> News? {
+        let now = Date()
+        if let firstStory = NewsManager.news.filter({ $0.date > now }).last ?? NewsManager.news.first {
```

Comment:
Other suggestion. Take the news array as an optional parameter defaulting to NewsManager.news.

Because I feel like we have this might need this for filtered subsets from: https://github.com/TCA-Team/iOS/issues/43

```Swift
func getNextUpcomingNews(in news: [News] = NewsManager.news) { ... }
```